### PR TITLE
Add list-usages command for container app

### PR DESCRIPTION
**Pull request by Juliehzl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/containerapp/azext_containerapp/custom.py [hunk 1]
```diff
@@ -49,7 +49,7 @@
     ContainerAppsJobClient,
     ContainerAppPreviewClient,
     AuthPreviewClient,
-    SubscriptionPreviewClient
+    SubscriptionPreviewClient, ManagedEnvironmentPreviewClient
```
Comment (Greedygre) [inline src/containerapp/azext_containerapp/custom.py:52]:
```suggestion
    SubscriptionPreviewClient, 
    ManagedEnvironmentPreviewClient
```

File: src/containerapp/azext_containerapp/commands.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
                             transform_job_execution_list_output,
                             transform_job_execution_show_output,
                             transform_revision_list_output,
-                            transform_revision_output)
+                            transform_revision_output, transform_usages_output)
```
Comment (Greedygre) [inline src/containerapp/azext_containerapp/commands.py:16]:
```suggestion
                            transform_revision_output, 
                            transform_usages_output)
```

**Review by Greedygre — APPROVED** [review]

File: src/containerapp/azext_containerapp/commands.py (hunk 2 above)
Comment (Juliehzl) [reply src/containerapp/azext_containerapp/commands.py:16]: current code also aligns with pylint check. I think it is fine.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**